Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserThumbnailView.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <AzCore/Memory/SystemAllocator.h>
+
+#include <QItemSelection>
+#include <QWidget>
+
+#endif
+
+namespace AzQtComponents
+{
+    class AssetFolderThumbnailView;
+}
+
+namespace AzToolsFramework
+{
+    namespace AssetBrowser
+    {
+        class AssetBrowserFilterModel;
+        class AssetBrowserTreeView;
+        class AssetBrowserThumbnailViewProxyModel;
+        class PreviewerFrame;
+
+        class AssetBrowserThumbnailView : public QWidget
```

Comment:
Maybe we can start a quest for undoing the smurf-naming here and call this class just `ThumbnailView`? It's in the `AssetBrowser` namespace already. Similar with `AssetBrowserThumbnailViewProxyModel`

Reply:
As we discussed on chat, I agree but would rather have a separate change globally unsmurfing the whole `AssetBrowser` subfolder.

Reply:
👍